packages/client/yggdrasil/yggdrasil.spec

```diff
@@ -68,9 +73,16 @@ make PREFIX=%{_prefix} \
 %{_datadir}/bash-completion/completions/*
 %{_mandir}/man1/*
 %{_prefix}/share/pkgconfig/%{name}.pc
+%if 0%{?suse_version}
+%{_bindir}/%{name}
+%else
 %{_libexecdir}/%{name}
+%endif
```

Comment:
where does this difference come from? Won't the systemd service still expect the binary to live in libexec?

Reply:
I have reverted this change. It is due to different paths on OpenSUSE/SLES for `/usr/lib`.